src/core/dom/jsx.js

```diff
@@ -82,7 +82,15 @@ export function createElement(tag, props, ...children) {
   if (typeof tag !== 'string') {
     return tag({...props, children});
   }
-  const element = self.document.createElement(tag);
+  // We expect the __svg magic prop to be set during compilation time.
```

Comment:
What do you mean by the opposite direction?

Reply:
Oooooh I get it now. Tracking #36733

Reply:
Actually, [according to this:](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/foreignObject)
```html
    <!--
      In the context of SVG embedded in an HTML document, the XHTML
      namespace could be omitted, but it is mandatory in the
      context of an SVG document
    -->
```

If we modify the transform to set `{'xmlns': 'http://www.w3.org/2000/svg'}` rather than `{__svg: true}`, we are able to support `foreignObject` transparently. Not only that, but by repeating the `xmlns` value throughout a bundle we're able to compress it better!